docs/source/app_developers_guide/aws.rst

```diff
@@ -0,0 +1,421 @@
+on-chain settings, along with a settings family transaction processor written
+in Python.
+
+One of the on-chain settings is the list of supported transaction families.
+You can configure this setting with a single command:
+
+In the example below, a JSON array is submitted to the `sawtooth config`
+command, which creates and submits a batch of transactions containing the
+settings change.
+
+The JSON array used tells the validator or validator network to accept
+transactions of the following types:
+
+* intkey
+* sawtooth_settings
+
+To create and submit the batch containing the new settings, enter the
+following commands from the Linux CLI:
+
+.. code-block:: console
+
+  $ sawtooth config proposal create sawtooth.validator.transaction_families='[{"family": "intkey", "version": "1.0", "encoding": "application/cbor"}, {"family":"sawtooth_settings", "version":"1.0", "encoding":"application/protobuf"}]'
+
+A TP_PROCESS_REQUEST message appears in the logging output of the validator,
+and output similar to the following appears in the validator logs:
+
+.. code-block:: console
+
+  [21:11:55.356 [Thread-9] tp_state_handlers DEBUG] GET: [('000000a87cb5eafdcca6a8cde0fb0dec1400c5ab274474a6aa82c12840f169a04216b7',b'\nl\n&sawtooth.settings.vote.authorized_keys\x12B03e3ccf73dd618ef1abe18da84d3cf5838a5d292d36ef8857a60b5ad04fd4ab517')]
+  [21:11:55.356 [Thread-9] interconnect DEBUG] ServerThread sending TP_STATE_GET_RESPONSE to b'afb61daaa87a4c70'
+  [21:11:55.362 [InterconnectThread-1] interconnect DEBUG] ServerThread receiving TP_STATE_GET_REQUEST message: 177 bytes
+  [21:11:55.371 [InterconnectThread-1] interconnect DEBUG] message round trip: TP_PROCESS_RESPONSE 0.021718978881835938
+  [21:11:55.373 [Thread-23] chain INFO] on_block_validated: 5da8c003(12, S:eb09cdf9, P:dab828cd)
+  [21:11:55.374 [Thread-23] chain INFO] Chain head updated to: 5da8c003(12, S:eb09cdf9, P:dab828cd)
+  [21:11:55.374 [Thread-23] publisher INFO] Now building on top of block: 5da8c003(12, S:eb09cdf9, P:dab828cd)
+  [21:11:55.375 [Thread-23] chain DEBUG] Verify descendant blocks: 5da8c003(12, S:eb09cdf9, P:dab828cd) ([])
+  [21:11:55.375 [Thread-23] state_delta_processor DEBUG] Publishing state delta from 5da8c003(12, S:eb09cdf9, P:dab828cd)
+  [21:11:55.376 [Thread-23] chain INFO] Finished block validation of: 5da8c003(12, S:eb09cdf9, P:dab828cd)
+
+
+You can verify that the settings change was successfully applied by
+checking the output of ``sawtooth config settings list``
+
+.. code-block:: console
+
+  $ sawtooth config settings list
+  sawtooth.settings.vote.authorized_keys: 03e3ccf73dd618ef1abe18da84d3cf5838a5d292d36ef8857a60b5ad04fd4ab517
+  sawtooth.validator.transaction_families: [{"family": "intkey", "version": "1.0", "encoding": "application/cbor"}, {"family":"sawtooth_settings", "version":"1.0", "e...
+
+
+Sawtooth Setup Service
+======================
+
+The Sawtooth AMI has a systemd service installed that handles some environment
```

Comment:
Could "The Sawtooth AMI has" be reworded to spell out AMI?  (It would improve web hits, plus just in case someone doesn't know.) Perhaps like this: 

     The Amazon Machine Image (AMI) for Sawtooth has... 

Also, I'd recommend saying either "has a systemd service that ..." or "installs a systemd service that...". (That is, avoid "has ... installed".)